deployment/appserver_config/oidc/conf.json

```diff
@@ -4,7 +4,7 @@
     },
     "clients": {
         "iqgeo-oidc": {
-            "issuer": "http://keycloak:8080/realms/iqgeo",
+            "issuer": "http://keycloak.local:8080/realms/iqgeo",
```

Comment:
maybe just push this change to master in case this PR takes a bit longer to merge